test/decl/nested/protocol.swift

```diff
@@ -32,7 +32,7 @@ struct ConformsToOuterProtocol : OuterProtocol {
   typealias Hen = Int
 
   func f() { let _ = InnerProtocol.self }
-  // expected-error@-1 {{use of unresolved identifier 'InnerProtocol'}}
+  // expected-error@-1 {{protocol 'InnerProtocol' can only be used as a generic constraint because it has Self or associated type requirements}}
```

Comment:
@slavapestov This is the only test change caused by adding nested types to result set in `LookupResultBuilder`.

Reply:
Excellent, thanks for following up!

Reply:
That’s for the help!